src/app/onion/learning-object-builder/components/content-upload/app/upload/upload.component.ts

```diff
@@ -203,17 +205,24 @@ export class UploadComponent implements OnInit, AfterViewInit, OnDestroy {
         if (object) {
           this.learningObjectCuid = object.cuid;
           this.notes = object.materials.notes;
-          this.bucketUploadPath = `${object.author.username}/${object.id}`;
-          this.files$.next(object.materials.files);
+
+          // Temporarily store the learning obejct to test if it is being updated
+          this.learningObject = object;
```

Comment:
I bet if we looked at the amount of memory we use in this builder to store the learning object a bunch of times we would be really sad.

Reply:
... yeahhhhhh